Apps/frontend/cypress/e2e/4-control-panel-bottom/amplitude-slider.cy.js

```diff
@@ -12,14 +12,14 @@ describe("amplitudeSlider", () => {
   it(`has label that displays reciprocal value "Infinity"`, () => {
     cy.get(`[data-cy="amplitudeSlider-0"]`).within(() => {
       cy.get('[type="range"]').invoke("val", 2).trigger("change");
-      cy.get("span").should("have.text", "0.50");
+      cy.get("div").should("have.text", "0.50");
```

Comment:
Cross-dependent with a few comments further down.

Reply:
As described changed to missbehavoir